server/src/librelay/relay_server_test.go

```diff
@@ -545,18 +546,18 @@ func TestReuseNonceOnDevMode(t *testing.T) {
 	// Clean up for the next test
 	relay.DevMode = false
 }
-
+/*
```

Comment:
If you think it's not meaningful, just remove it.

Reply:
It is a go server test... Will remove it soon enough I hope :-)